### ISSUE TITLE
3 tests failed. diff:diff across DST (2336.17) diff:diff across DST (2336.19) relative time:custom rounding (2630.3) 

### ISSUE THREAD

**Issue by mokingguy** [moment/moment#3555]
### Client info
```
Date String   : Tue Nov 01 2016 13:58:48 GMT-0600 (CST)
Locale String : 11/1/2016, 1:58:48 PM
Offset        : 360
User Agent    : Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.71 Safari/537.36
Moment Version: 2.15.2
```

====

### diff:diff across DST (2336.17) 

month diff across DST, lower bound

```javascript
// Expected true
// Actual   false
false === true
```

====

### diff:diff across DST (2336.19) 

year diff across DST, lower bound

```javascript
// Expected true
// Actual   false
false === true
```

====

### relative time:custom rounding (2630.3) 

Round down towards the nearest day

```javascript
// Expected in 30 days
// Actual   in a month
"in a month" === "in 30 days"
```

**Comment by mj1856** [comment]
See #3021

**Comment by mj1856** [comment]
FYI, the 2630.3 issue was just with the unit test itself, and was fixed with faa85ed0a517bbca3b14d442655aff925e5b8209 to be deployed in the next version (2.15.3 or 2.16.0).
